src/Cli/dotnet/commands/dotnet-workload/install/LocalizableStrings.resx

```diff
@@ -330,4 +330,7 @@
   <data name="CheckForUpdatedWorkloadManifests" xml:space="preserve">
     <value>Checking for updated workload manifests.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="InvalidVersionForWorkload" xml:space="preserve">
+    <value>Error parsing version for workload {0}: Invalid version {1}.</value>
```

Comment:
```suggestion
    <value>Error parsing version `{1}` for workload manifest ID '{0}'</value>
```

@baronfel Any suggestions for the wording / formatting of this message?

Reply:
In lieu of any failure reasons for the parsing failure, I think your suggestion is good. It read more naturally to me. The biggest concern I had was making sure the workload Id was available in the error message, so users had better guidance to the root cause. I would suggest using single-quotes instead of backticks in the message, though.